pages/api/v1/answers.ts

```diff
@@ -468,6 +467,18 @@ async function Angela( // Good ol' Angela
   // When this number is reached, we remove the oldest messages from the context window
   const maxConvLength = model === "gpt-4-0613" ? 20 : 14;
 
+  // Store the system prompt in the feedback table so we can reconstruct it later
+  const { data, error } = await supabase.from("feedback").insert({
+    system_prompt: getMessages(
+      [],
+      actions,
+      reqData.user_description,
+      org,
+      language
+    )[0].content,
+    conversation_id: conversationId,
+  });
```

Comment:
I don't like this - it means we store lots more system prompts (~4kB a pop) than we need

Can we instead store system prompts every time feedback is given?